rules/explain-deleted-or-modified-appointments/rule.md

```diff
@@ -22,39 +22,41 @@ created: 2010-03-15T08:42:32.000Z
 archivedreason: null
 guid: 1c278726-9df4-44dc-b09b-bb17470c31ad
 ---
-Every change to an appointment is a hassle, and it's good to be explicit as to why the appointment had to change, so always give a reason. Unlike an email, changing an appointment will not have the original content recorded, it is a good idea to make it visible by adding the version number and date of changes to the history. This shows the timeline of the changes and how many changes they've asked for! When we have v7, people will start questioning how this has happened.
+Every change to an appointment is a hassle, and it's good to be explicit as to why the appointment had to change, so always give a reason. 
+
+Unlike an email, changing an appointment will not have the original content recorded, it is a good idea to make it visible by adding the version number and date of changes to the history. This shows the timeline of the changes and how many changes they've asked for! When we have v7, people will start questioning how this has happened.
 
 Whenever you have to modify the time, attendees, or subject of an appointment, always send the updated appointment with comments at the top to explain why you want to make these changes.
 
 There are two key ways to do this:
 
 1. Version number - Adding the v# to the notes can help people to track all of the changes. 
 
-     E.g. "**v2** - Updated 27/03: Changed time to 04/04 9:00am because Bob can not make it"
+     E.g. "**V2** - Updated 27/03: Changed time to 04/04 9:00am because Bob can not make it"
      
 2. Actions words and date of change - These are prepended to any notes that you add to the top of the appointment so that people can "at-a-glance", understand what's going on with the change and when the changes are happened.
 
-     E.g. "v3 - **Updated 30/03**: Changed time to 30/03 10:00am because Bob can not make it again"
+     E.g. "V3 - **Updated 30/03**: Changed time to 07/04 10:00am because Bob can not make it again"
```

Comment:
Hey @JimmyChenSSW 
Any reason to change the date here?
I suggest keeping the day more than 12, so it cannot be confused with American way to have dates (e.g 4 th of July OR 7th of April?)... where months come 1st.

Reply:
Hey @tiagov8 
That is a very good point. I changed the date to make it realistic. I can and will try always to give the example of a date over the 12th.